src/GameView.cpp

```diff
@@ -44,6 +47,16 @@ void GameView::drawText(sf::Text &text) {
 	this -> App.draw(text);
 }
 
+/*
+* Draws numeric text for the level
+* @param text: text to draw
+*/
+void GameView::drawNumText(sf::Text &text) {
+	text.setFont(*this->numFont);
+	text.setFillColor(sf::Color::White);
+	this->App.draw(text);
```

Comment:
Add a space between this & App (this -> App). Use this spacing convention for the rest of your code as well.